### PR TITLE
Order Plugin List alphabetically

### PR DESCRIPTION
**Pull request by pivilartisant**
we order alphabetically our plugins list

### CODE REVIEW

File: web/massastation/src/pages/Store/StationSection/StationSection.tsx [hunk 1]
```diff
@@ -39,6 +39,17 @@ function StationSection({
...
+  // Sorting the plugins array alphabetically by plugin name
+  const sortedPlugins = plugins?.sort((a, b) => {
+    if (a.name < b.name) {
+      return -1;
+    }
+    if (a.name > b.name) {
+      return 1;
+    }
+    return 0;
+  });
```
Comment (mazmassa) [inline web/massastation/src/pages/Store/StationSection/StationSection.tsx:51]: What we are sorting here!?
Object, array of strings or numbers!? 🤔 

Isn't that enought?
```ts
my-object.sort((a, b) => a.age - b.age);
```

Comment (pivilartisant) [reply web/massastation/src/pages/Store/StationSection/StationSection.tsx:51]: an array of ojbects

Comment (pivilartisant) [reply web/massastation/src/pages/Store/StationSection/StationSection.tsx:51]: > What we are sorting here!? Object, array of strings or numbers!? thinking
> 
> Isn't that enought?
> 
> ```ts
> my-object.sort((a, b) => a.age - b.age);
> ```

I get this error with this suggestion, I do not think it is worth converting type to achieve this

The left-hand side of an arithmetic operation must be of type 'any', 'number', 'bigint' or an enum type.ts(2362)

Comment (Pierregvx) [reply web/massastation/src/pages/Store/StationSection/StationSection.tsx:51]: what if you parse int ?

Comment (pivilartisant) [reply web/massastation/src/pages/Store/StationSection/StationSection.tsx]: we found  a more efficient way :)

File: web/massastation/src/pages/Store/StoreSection/StoreSection.tsx [hunk 2]
```diff
@@ -47,6 +47,8 @@ function StoreSection({
     );
   };
 
+  const sortedPlugins = plugins?.sort((a, b) => a.name.localeCompare(b.name));
```
Comment (mazmassa) [inline web/massastation/src/pages/Store/StoreSection/StoreSection.tsx:50]: Last: can we have that in a classical function?

**Review by mazmassa — APPROVED** [review]

File: web/massastation/src/pages/Store/StoreSection/StoreSection.tsx (hunk 2 above)
Comment (pivilartisant) [reply web/massastation/src/pages/Store/StoreSection/StoreSection.tsx:50]: ofc

File: web/massastation/src/pages/Store/StationSection/StationSection.tsx [hunk 3]
```diff
@@ -49,7 +52,7 @@ function StationSection({
         <>
           {plugins && plugins.length ? (
             <div className="flex gap-4 flex-wrap">
-              {plugins.map((plugin, index: number) => (
+              {sortPlugins(plugins)?.map((plugin, index) => (
```
Comment (Thykof) [inline web/massastation/src/pages/Store/StationSection/StationSection.tsx:55]: Are you sure you need the "?"

**Pull request merged**